client/src/lib/utils/Application/RemoveApplication.ts

```
@@ -1,22 +1,30 @@
-import {SupabaseClient, PostgrestError} from '@supabase/supabase-js';
-import {Profile} from '../../store/profile';
-import {createSupabaseClient} from '../../supabase/client';
+import { SupabaseClient, PostgrestError } from "@supabase/supabase-js";
+import { Profile } from "../../store/profile";
+import { createSupabaseClient } from "../../supabase/client";
 
 export async function RemoveApplication(
-    profile: Profile,
-    applicationId: string,
-    supabase: SupabaseClient = createSupabaseClient()
+  profile: Profile,
+  applicationId: string,
+  supabase: SupabaseClient = createSupabaseClient()
 ): Promise<{ success: boolean | null; error: PostgrestError | null }> {
-    const {error} = await supabase
-        .from('application')
-        .delete()
-        .eq('application_id', applicationId)
-        .eq('profile_id', profile.profile_id);
+  const { error } = await supabase
+    .from("application")
+    .delete()
+    .eq("application_id", applicationId)
+    .eq("profile_id", profile.profile_id);
 
-    if (error) {
-        console.error('Error:', error.message);
-        return {success: false, error};
-    }
-    return {success: true, error: {message: 'No data returned'} as PostgrestError};
-    
-}
\ No newline at end of file
+  if (error) {
+    console.error("Error:", error.message);
+    return {
```

why not just return the error object? I would just return the error object here